target/README.md

```diff
@@ -69,28 +74,34 @@ The correlation configuration is used to specify which properties will be used t
+The following features are available:
+
+| Feature                                   | Supported | Actions                                 | Remarks           |
+| ----------------------------------------- | --------- | --------------------------------------- | ----------------- |
+| **Account Lifecycle**                     | ✅        | Create, Update, Enable, Disable, Delete |                   |
+| **Permissions**                           | ✅        | Retrieve, Grant, Revoke                 | Static or Dynamic |
+| **Resources**                             | ❌        | -                                       |                   |
+| **Entitlement Import: Accounts**          | ✅        | -                                       |                   |
+| **Entitlement Import: Permissions**       | ❌        | -                                       |                   |
+| **Governance Reconciliation Resolutions** | ✅        | -                                       |                   |
 
-| Action                                  | Description                                                                                 |
-| --------------------------------------- | ------------------------------------------------------------------------------------------- |
-| create.ps1                              | Creates a new account.                                                                      |
-| delete.ps1                              | Removes an existing account or entity.                                                     |
-| disable.ps1                             | Disables an account, preventing access without permanent removal.                           |
-| enable.ps1                              | Enables an account, granting access.                                                       |
-| update.ps1                              | Updates the attributes of an account.                                                      |
-| permissions/groups/grantPermission.ps1  | Grants specific permissions to an account.                                            |
-| permissions/groups/revokePermission.ps1 | Revokes specific permissions from an account.                                         |
-| permissions/groups/permissions.ps1      | Retrieves all available permissions.                                                       |
-| resources/groups/resources.ps1          | Manages resources, such as creating groups.                                                |
-| configuration.json                      | Contains the connection settings and general configuration for the connector.              |
-| fieldMapping.json                       | Defines mappings between person fields and target system person account fields.              |
 
 ### Field mapping
 
 The field mapping can be imported by using the _fieldMapping.json_ file.
 
+### Account Reference
+
+The account reference is structured as follows:
```

Comment:
@RudolfAmersfoort Is dat wat jij ongeveer in gedachten had m.b.t. de account reference?

Reply:
Besproken met Mauro, en uitgelegd wat ik bedoelde